src/pages/components/table-of-contents.mdx

```diff
@@ -32,7 +32,16 @@ The Table of Contents (ToC) component renders as left navigation that includes a
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+![Image of Table of contents component usecase](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+
+**When to use:**
+* When the content of the page is long or text heavy and will require a lot of scrolling.
```

Comment:
Small suggestion here

```suggestion
* When the page length is long and requires a lot of scrolling.
```

Reply:
@photodow I get what you say, but will the word "length" enough to tell adopters we are talking about the content's length?

Reply:
I don't think we're talking about content's length though. This component is not determined by the amount of content on the page, but instead by how far down the page the content might be.

Reply:
Good point. James.